spec/models/game_spec.rb

```diff
@@ -2,12 +2,22 @@
 
 RSpec.describe Game do
   it { is_expected.to have_attribute :name }
+  it { is_expected.to validate_presence_of :name }
 
   it { is_expected.to belong_to :white_player }
-
   it { is_expected.to belong_to :black_player }
 
   it { is_expected.to have_many :chess_pieces }
 
-  it { is_expected.to validate_presence_of :name }
+  describe '#create' do
+    context 'when initializing the state of the board' do
+      it 'puts a Black King in the correct starting position' do
+        game = create :game
+        black_king = game.chess_pieces.where(position_x: 4, position_y: 0).first
```

Comment:
an alternative syntax for this would be

`game.chess_pieces.find_by(position_x: 4, position_y: 0)`